ci-operator/step-registry/lvms/upgrade/lvms-upgrade-commands.sh

```
@@ -0,0 +1,119 @@
+
+    local output
+    if ! output=$(oc patch subscription lvms-operator -n "${LVMS_NAMESPACE}" --type=merge \
+        -p "{\"spec\":{\"channel\":\"${TARGET_LVMS_CHANNEL}\",\"installPlanApproval\":\"Manual\"}}" 2>&1); then
+        echo "ERROR: Failed to update subscription: ${output}"
+        return 1
+    fi
+}
+
+function wait_for_installplan() {
+    local old_installplan
+    old_installplan=$(get_current_installplan)
```

Shellcheck (0.11.0)

[warning] old_installplan appears unused. Verify use (or export if used externally).

(SC2034)